### PR TITLE
Fixes issue#511. Added a button

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes issue #511 
Added a button similar to the other at the bottom of the contributors section.
Updated index.html

### CODE REVIEW

**Comment by NaviRocker** [comment]
Please produce a Preview Link.

**Comment by arjunpandey2003** [comment]
@naveentherocker my deploy link is not working here take a look https://arjunpandey2003.github.io/gci17.fossasiapcd.org/

**Comment by NaviRocker** [comment]
@arjunpandey2003 The Link is working but the above-mentioned changes are not visible.

**Comment by arjunpandey2003** [comment]
Thats what @naveentherocker

**Comment by the-codingninja** [comment]
Your preview link is not updated.
Please update it and your latest changes will show up in it then

File: index.html [hunk 1]
```diff
@@ -111,7 +111,8 @@ <h2 class="animated fadeIn">Contributors</h2>
 		<div class="col-sm-12 col-md-12 text-center">
 	<a  class="btn" data-toggle="popover" title="" data-placement="bottom" href="https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/">How can I be a contributor?</a>	
 	<a  class="btn" data-toggle="popover" title="" data-placement="bottom" href="https://github.com/fossasia/gci17.fossasia.org">View the Github Repository <i class="fa fa-github fa-lg"></i></a>
-	<a  class="btn" data-toggle="popover" title="" data-placement="bottom" href="https://gci16.fossasia.org/#contributors">Contributors Gci'16</a>		
+	<a  class="btn" data-toggle="popover" title="" data-placement="bottom" href="https://gci16.fossasia.org/#contributors">Contributors GCI'16</a>	
+	<a  class="btn" data-toggle="popover" title="Gci15" data-placement="bottom" href="https://gci15.fossasia.org/#contributors">Contributors GCI'15</a>
```
Comment (NaviRocker) [inline index.html]: I think it's better if you could add `target="_blank"` for every link.

**Review by ParthS007 — CHANGES_REQUESTED** [review]
Please do suggested changes and provide a preview link

**Comment by arjunpandey2003** [comment]
@ParthS007 i have made the changes but there is some problem with My preview link my changes are not being shown

**Comment by arjunpandey2003** [comment]
https://arjunpandey2003.github.io/gci17.fossasiarjunp.org/.

**Pull request closed without merging**